pyproject.toml

```diff
@@ -32,6 +32,8 @@ pygithub = "~=2.1.1"
 # extras: fastapi
 fastapi = { version = ">0.101", optional = true }
 uvicorn = { version = "^0.22", extras = ["standard"], optional = true }
+# extras: wss
+websocket-client = { version = "^1.7.0", optional = true }
```

Comment:
Is there a specific reason you chose this package?

Reply:
This was a package we chose a while back in a specific pipeline implementation and it has worked well since then. 

The project looks decently well maintained and has no dependencies. 

After more research it seems like the `websocket-client` library is probably a better choice for us over the other well maintained `websockets` library given it allows flexibility to work with both synchronous and asynchronous scenarios. A major benefit for `websockets` is that there is less boilerplate for managing simultaneous connections. In our scenario, we'd manage simultaneous connections across Aineko nodes, so that is not a concern.

Reply:
Thanks, I appreciate the well thought out response. 🙏